app/templates/gulp/istanbul.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var gulp   = require('gulp');
+var plugins = require('gulp-load-plugins')();
+
+module.exports = function (paths, config) {
+  gulp.task('istanbul', function (cb) {
+    gulp.src(paths.source)
+      .pipe(plugins.istanbul()) // Covering files
+      .pipe(plugins.istanbul.hookRequire()) // Force `require` to return covered files
+      .on('finish', function () {
+        gulp.src(paths.tests)
+          .pipe(plugins.plumber(config.plumber))<% if (testFramework === 'jasmine') { %>
```

Comment:
Line is too long.<br>Expected an identifier and instead saw '%'.<br>Unrecoverable syntax error. (56% scanned).